src/widgets/menu/__tests__/__snapshots__/menu-test.ts.snap

```diff
@@ -17,20 +17,7 @@ Object {
     "showMore": "ais-Menu-showMore",
   },
   "depth": 0,
-  "facetValues": Array [
-    Object {
-      "label": "foo",
-      "transformed": true,
-      "url": "#",
-      "value": undefined,
-    },
-    Object {
-      "label": "bar",
-      "transformed": true,
-      "url": "#",
-      "value": undefined,
-    },
-  ],
+  "facetValues": Array [],
```

Comment:
todo: find which change in the tests caused this to change (likely something using the ts-safe result constructors